cmd/infracost/comment_bitbucket.go

```diff
@@ -137,6 +140,7 @@ func commentBitbucketCmd(ctx *config.RunContext) *cobra.Command {
 	cmd.Flags().Var(&prNumber, "pull-request", "Pull request number to post comment on")
 	cmd.Flags().String("repo", "", "Repository in format workspace/repo")
 	_ = cmd.MarkFlagRequired("repo")
+	cmd.Flags().Bool("summary-only", false, "Omit diff details so only the summary table is output")
```

Comment:
@alikhajeh1 Guess we could rephrase this description with the new flag `--exclude-cli-output`. Maybe something like `Do not include CLI output`? Not sure if negation works better than "exclude".

Reply:
good point, I added a suggestion.